packages/council-proposals/src/getProposals.ts

```diff
@@ -40,17 +42,31 @@ export async function getProposals(
 
       const snapshotId =
         snapshotIdsByProposalId[proposalId] ||
-        // Temporary: default to the first one if more proposals exist
-        // on-chain than are in the snapshot space,
-        snapshotIdsByProposalId[0];
+        // Set snapshotId to -1 for unverified proposals
+        "-1";
```

Comment:
Let's use `""` empty string instead to be more consistent with the other empty values in the proposals.json